packages/frontend/app/models/channel-context-chain.ts

```diff
@@ -1,13 +1,16 @@
-import Model, { attr, hasMany } from '@ember-data/model';
+import Model, { hasMany, belongsTo } from '@ember-data/model';
 
 import Identity from 'emberclear/models/identity';
 import VoteChain from './vote-chain';
+import Channel from './channel';
 
 export default class ChannelContextChain extends Model {
-  @attr() admin!: Identity;
-  @hasMany('identity', { async: false }) members!: Identity[];
-  @attr() supportingVote!: VoteChain;
-  @attr() previousChain!: ChannelContextChain;
+  @belongsTo('identity', { async: false, inverse: 'adminOf' }) admin!: Identity;
+  @hasMany('identity', { async: false, inverse: 'memberOf' }) members!: Identity[];
+  @belongsTo('vote-chain', { async: false }) supportingVote!: VoteChain;
+  @belongsTo('channel-context-chain', { async: false, inverse: 'parentChain' }) previousChain!: ChannelContextChain;
+  @belongsTo('channel-context-chain', { async: true, inverse: 'previousChain' }) parentChain!: ChannelContextChain;
```

Comment:
why `async: true`?

Reply:
We set async to be true for the records that we will never be accessing, so that it doesn't try to load them whenever we load the records.

Reply:
does the relationship need to be defined then? is the data ever loaded?

Reply:
I'm pretty sure that if you have to have the back relationships for them to be set up right, at least according to what I was reading in the ember docs earlier today.

Reply:
We were following https://guides.emberjs.com/release/models/relationships/

Reply:
Addressed in d005d5f